docs/community/bindings.md

```diff
@@ -2,6 +2,10 @@
 
 As a C library, bindings can be made to call H3 functions from different programming languages. This page lists different bindings currently available. Contributions to this list are welcome, please feel free to open a [pull request](https://github.com/uber/h3/tree/master/docs/community/bindings.md).
 
+## C# 
```

Comment:
```suggestion
## ``C#``
```

Unfortunately it looks like our Markdown parser thinks the `#` in `C#` is formatting and doesn't render it correctly. Could you replace with ` ``C#`` `?

Reply:
Hi Isaac, I've changed from C# to C-sharp, more markdown compatible. Sorry about that!